.github/workflows/conformance-datapath-v1.13.yaml

```diff
@@ -286,15 +286,15 @@ jobs:
           provision: 'false'
           cmd: |
             cd /host/
-            ./contrib/scripts/kind.sh "" 3 "" "" "${{ matrix.kube-proxy }}" "dual"
+            ./contrib/scripts/kind.sh --xdp "" 3 "" "" "${{ matrix.kube-proxy }}" "dual"
```

Comment:
Not sure if changing it for 1.13 is sensible?

Reply:
https://github.com/cilium/cilium/pull/24470 got merged, so the master failure should be fixed. We can change the v1.13 later on.

Anyway, I'd suggest to add `xdp: true` to the configuration matrix, and then conditionally to enable it.

Reply:
I don't see any downside enabling it unconditionally, so yeah I'm fine either way.

Reply:
This change in 1.13 workflows breaks datapath test in 1.13 backports: https://github.com/cilium/cilium/actions/runs/4544793510

Turns out backporting just this PR to fix it is not enough. Thus, I've sent a partial revert for now: https://github.com/cilium/cilium/pull/24611